### PR TITLE
[HOTFIX][RELEASE1.0] Fixed label_scheme mismatch in classification

### PR DESCRIPTION
**Pull request by harimkang**
## Summary
Fix classification label schema mismatch

A problem occurs because the class setting in incremental training and the class setting in the exporter are different. (Occurs when keep using the same Task object, can see it in the sample code.)

However, I'm not familiar with the `mpa/cls` code, so please take a look at @JihwanEom  and @supersoob  even after the merge.

### CODE REVIEW

File: otx/mpa/cls/incremental/stage.py [hunk 1]
```diff
@@ -35,35 +35,27 @@ def configure_task(self, cfg, training, **kwargs):
...
-        train_data_cfg = Stage.get_data_cfg(cfg, "train")
+        train_data_cfg = self.get_data_cfg(cfg, "train")
         if training:
             if train_data_cfg.type not in CLASS_INC_DATASET:
                 logger.warning(f"Class Incremental Learning for {train_data_cfg.type} is not yet supported!")
             if "new_classes" not in train_data_cfg:
                 logger.warning('"new_classes" should be defined for incremental learning w/ current model.')
 
             if cfg.model.type in WEIGHT_MIX_CLASSIFIER:
+                cfg.task_adapt.final = self.model_classes
```
Comment (eunwoosh) [inline otx/mpa/cls/incremental/stage.py]: I think `cfg.task_adapt.final` isn't used anywhere. How about deleting this line?

File: otx/mpa/cls/stage.py [hunk 2]
```diff
@@ -143,25 +143,28 @@ def configure_task(self, cfg, training, **kwargs):
...
+            model_classes = org_model_classes + [cls for cls in data_classes if cls not in org_model_classes]
+        else:
+            raise KeyError(f"{self.task_adapt_op} is not supported for task_adapt options!")
+
+        # Model architecture
+        cfg.model.head.num_classes = len(model_classes)
+
+        self.org_model_classes = org_model_classes
         self.model_classes = model_classes
         self.data_classes = data_classes
```
Comment (eunwoosh) [inline otx/mpa/cls/stage.py]: As above, this line isn't used anywhere, I think. How about deleting it?

**Review by eunwoosh — COMMENTED** [review]
LGTM in general. I submitted a few tiny comments. please take a look.

File: otx/mpa/cls/incremental/stage.py (hunk 1 above)
Comment (harimkang) [reply otx/mpa/cls/incremental/stage.py]: Done

File: otx/mpa/cls/stage.py (hunk 2 above)
Comment (harimkang) [reply otx/mpa/cls/stage.py]: Done

**Review by eunwoosh — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]
Nice catch!

**Pull request merged**